js/subscribe.js

```diff
@@ -0,0 +1,11 @@
+ 
+$(document).ready(function () {
+
+ $("#btn_subscribe").click(function () {
+        var txtEmail = $('#txtSubscribeEmail').val();        
+            var ajaxRequest = $.post(
+             "js/handlers/subscribe.php" ,{ email: txtEmail },function(data,status){
+    alert("Thank you for Subscribing");
```

Comment:
please change the message to "Thank you for subscribing.Please check your email to confirm"